src/types.js

```diff
@@ -3,3 +3,63 @@ export type Data = {
   key: string,
   value: string,
 };
+
+export type GlobalAttributes = $Shape<{|
+  hidden: boolean,
+  id: string,
+  style: { [key: string]: any },
```

Comment:
After our meeting: please replace `style` with `classNames` and make it optional.

Reply:
Aaaand done.

Reply:
By the way, since this is wrapped in `$Shape<>`, all these props are optional.